physical-based-animations.go

```diff
@@ -105,6 +104,13 @@ func run() {
 		max:   4,
 	}
 
+	initialVelocity := Parameter{
+		value: 9.5,
+		step:  0.5,
+		min:   -2,
```

Comment:
Why the `min` value is negative? Is that making a sense?

Reply:
That's just for fun. It creates a nice visual display of the functionality and actually works properly.